src/main/java/rs/hybridit/repository/BookInstanceRepository.java

```diff
@@ -0,0 +1,11 @@
+package rs.hybridit.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import rs.hybridit.model.BookInstance;
+
+@Repository
```

Comment:
maybe extend PagingAndSortingRepository so you can do paging since a number of books can be large